tests/__init__.py

```diff
@@ -277,6 +280,10 @@ def test_smallest(self):
             ("ssh", "-V"),
             # We are able to dump
             ("pg_dump", "-f/var/lib/odoo/prod.sql", "prod"),
+            # Geoip should not be activated
+            ("bash", "-xc", 'test "$(which geoipupdate)" != ""'),
```

Comment:
```suggestion
            ("which", "geoipupdate"),
```

Reply:
The difference between the two lines is that bash -xc also prints the invoked command making it easier to debug if it fails.